helpers/pyband/pyband/wallet.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import annotations
+
+import hashlib
+
+from bech32 import bech32_encode, convertbits
+from bip32 import BIP32
+from ecdsa import SigningKey, VerifyingKey, SECP256k1
+from mnemonic import Mnemonic
+
+
+class PrivateKey:
+    def __init__(self, signing_key: SigningKey) -> None:
+        self.signing_key = signing_key
+
+    @classmethod
+    def from_mnemonic(cls, words: str, path="m/44'/494'/0'/0/0") -> PrivateKey:
+        seed = Mnemonic("english").to_seed(words)
+        return cls(
+            SigningKey.from_string(
+                BIP32.from_seed(seed).get_privkey_from_path(path),
+                curve=SECP256k1,
+                hashfunc=hashlib.sha256,
+            )
+        )
+
+    def to_pubkey(self) -> PublicKey:
+        return PublicKey(self.signing_key.get_verifying_key())
+
+    def sign(self, msg: bytes) -> bytes:
+        pass
+
+
+class PublicKey:
+    def __init__(self, verify_key: VerifyingKey) -> None:
+        self.verify_key = verify_key
+
+    @classmethod
+    def from_bech32(cls, bech: str, prefix="bandvalconspub") -> PublicKey:
+        pass
+
+    def __str__(self) -> str:
+        return self.verify_key.to_string("compressed")
+
+    def to_bech32(self, prefix="bandvalconspub") -> str:
```

Comment:
Same here. I think simpler to just have 3 functions. We can have an internal helper function.